cloud-aws/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsRdsDbParameterGroupViewTest.java

```diff
@@ -13,6 +19,8 @@
 
 class AwsRdsDbParameterGroupViewTest {
 
+    public static final String POSTGRES_10 = "postgres10";
```

Comment:
Perhaps should be `private`. Furthermore, this constant is not used anywhere; see below.

Reply:
I moved a test into the parametrized test and forgot to remove this.